Sources/MapLibreSwiftUI/Examples/Camera.swift

```diff
@@ -10,7 +10,8 @@ struct CameraDirectManipulationPreview: View {
 
     var body: some View {
         MapView(styleURL: styleURL, camera: $camera)
-            .onStyleLoaded { _ in
+            .onMapStyleLoaded { _ in
+                print("Style Loaded")
```

Comment:
Not sure if this print was left in on purpose. While I try to avoid prints, especially in libs, this is only part of our example, so no hard opinion on this.